src/components/pill-selector-dropdown/RoundPill.scss

```diff
@@ -59,47 +37,11 @@
 }
 
 .bdl-RoundPill--warning {
-    background-color: $bdl-yellorange-10;
-    border: 1px solid $bdl-yellorange-50;
-
-    .bdl-Avatar-externalBadge {
-        border-color: $bdl-yellorange-10;
-    }
+    @include bdl-RoundPill($background-color: $bdl-yellorange-10, $border-color: $bdl-yellorange-50, $selected-background-color: $bdl-yellorange-20, $selected-border-color: $bdl-yellorange);
```

Comment:
I had each parameter on its own line, but it looks like the code gets automatically reformatted either when committing or when pushing the code.